### DISCUSSION TITLE
User cannot modify a password

### DISCUSSION THREAD

**Discussion by G-eos** [dani-garcia/vaultwarden#3218] [Q&A] [answered]
### Subject of the issue
One of the user cannot modify a password already recorded in its own vault. Either through plugin to browser or directly on the web interface.
Logs: say:
[2023-02-02 13:57:15.887][vaultwarden::api::core::ciphers][ERROR] Due to an Enterprise Policy, you are restricted from saving items to your personal vault.

This is probably a configuration issue, but I cannot figure where :-(

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.27.0
* Web-vault version: v2022.12.0
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true

### Steps to reproduce
connect to web interface, select an item. Try to change password.

### Expected behaviour
Change password

### Actual behaviour
Raise error: Due to an Enterprise Policy, you are restricted from saving items to your personal vault.
### Troubleshooting data
[2023-02-02 13:57:15.887][vaultwarden::api::core::ciphers][ERROR] Due to an Enterprise Policy, you are restricted from saving items to your personal vault.

Thanks !

**Comment by tessus** [comment, marked as answer]
This is not a bug.

The admin of the organization which that user joined set the following org policy:

```
Remove individual vault
Require members to save items to an organization by removing the individual vault option.
```

This means that all users that join this org cannot have a personal vault. I am not sure, but I think it shows somewhere which policies are set.
IMO, the policies should be listed when a user is invited to an org. I am not sure if this is done. If not, maybe you can request this feature at the Bitwarden forum.
I am not sure if this project implements features which are not in the upstream project.

P.S.: I am not sure if "upstream" is the right term in this case. I meant the official Bitwarden server.

**Reply by G-eos** [reply]
Thanks ! And its working for my second user as it is an administrator and not for this one !
